website/src/install.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
```

Comment:
Very cool. Next upgrade I need to do, I'll try this out.

Reply:
Please do, I have only been able to test this on Linux, mac, Win on Intel hardware.